srcpkgs/reuse/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'reuse'
+pkgname=reuse
+version=0.13.0
+revision=1
+build_style=python3-module
+make_check_args="-k not((test_lint_read_errors)or(test_generate_project_report_read_error))"
```

Comment:
`--deselect=` should also work and it will be more readable

Reply:
Thanks!